.gitignore

```diff
@@ -69,3 +69,5 @@ pom.xml
 
 /bazel*
 .bazelrc
+/cclient/bazel*
```

Comment:
just remove the `/` before the `/bazel*` instead of adding the cclient line
same goes for cmake